src/ConcreteOperations/interior.jl

```diff
@@ -0,0 +1,28 @@
+export is_interior_point
```

Comment:
You can just put this code in [`LazySet.jl`](https://github.com/JuliaReach/LazySets.jl/blob/master/src/Interfaces/LazySet.jl).

Reply:
Ah, sorry, that was ambiguous. I meant the whole function.